core/lib/assert.c

```diff
@@ -27,7 +27,7 @@ __NORETURN void _assert_failure(const char *file, unsigned line)
 
 __NORETURN void _assert_panic(void)
 {
-    cpu_print_last_instruction();
+    printf("%p\n", (void *)cpu_get_last_instruction());
```

Comment:
```suggestion
    printf("%p\n", (void *)cpu_get_caller_pc());
```
the `_last_instruction` part seems to be a bit misleading

(see other discussions)